governance/roles.md

```diff
@@ -81,12 +83,61 @@ Project Leads are nominated and approved by the following process:
   to ensure that the Project Lead is set up for success or suggest alternatives.
 
 
-## TOC Liaison
+## TOC liaison
 
 The [CNCF SIG](https://github.com/cncf/toc/blob/master/sigs) process identifies
 a TOC Liaison.  The SIG chairs are responsible for establishing effective
 communication with the TOC liaison, including further communication to the
 wider TOC upon request.
 
 The TOC Liaison will occasionally prioritize SIG activities, as needed by the
-TOC, to further the [CNCF mission](https://github.com/cncf/foundation/blob/master/charter.md#1-mission-of-the-cloud-native-computing-foundation).
\ No newline at end of file
+TOC, to further the [CNCF mission](https://github.com/cncf/foundation/blob/master/charter.md#1-mission-of-the-cloud-native-computing-foundation).
+
+
+## Facilitation roles
+
+Facilitation roles are identified in [github settings](/.github/settings.yaml)
+and include additional permissions in the repo which are governed by policies
+described below.
+
+### Security Assessment Facilitator
+
+[Security Assessments](/assessments) are part of the on-going work of the group
+and led by a Security Assessment Facilitator, who will:
+
+* coordinate security review leads for upcoming security assessments.
+* identify and recommend security reviewers.
+* contribute to process improvements.
+* review and merge PRs in the /assessments directory (ensuring co-chair review
```

Comment:
I don't see permissions assigned to this role above

Reply:
*this* is the specification of the permissions -- it's an attempt to scope what this person should do (and describe the required reviews)

Reply:
I meant above,. If they are meant to merge in PRs they aren't listed as push or a corresponding role with humans and permission to push.  It currently implies the assessment facilitator can only be one of the previous humans with existing push.